modules/service/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -2921,13 +2883,15 @@ enum CloudExtinction {
   THREE_POINT_ZERO
 }
 
-# ConfigurationMode
-enum ConfigurationModeType {
-  # ConfigurationModeType GmosNorthLongSlit
+# ObservingMode
+enum ObservingModeType {
```

Comment:
The old "science mode" and "configuration mode type" would become "observing mode" and "observing mode type".